.isort.cfg

```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=
+known_third_party=setuptools
```

Comment:
this change shouldn't be here

Reply:
absolutely @i-vyshnevska .. but then the travis checks fails.. did you try to commit with current 13.0/travis configuration?

Reply:
ok, push this in separate commit pls

Reply:
@pegonzalezspesol  sorry, I wasn't very explicit, 
that change about issort is correct, you can open other PR or push in this but is separate commit so migration commits are not mixup with travis setup

Reply:
done @i-vyshnevska